shopinvader_schema_invoice/schemas/invoice.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Camptocamp SA
+# @author: Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+from .amount import InvoiceAmount
+
+
+class Invoice(StrictExtendableBaseModel, extra="ignore"):
+    id: int
+    name: str
+    date_invoice: date
+    date_due: date | None = None
+    state: str
```

Comment:
```suggestion
    state: Literal["open", ...]
```

I've not checked recently if FastAPI supports Literal, but if it does this is good documentation value.

Reply:
FastAPI supports Literal but also Enums.  In my project I use enums.